src/tools/rbd/action/BenchWrite.cc

```diff
@@ -262,6 +262,11 @@ int execute(const po::variables_map &vm) {
   } else {
     bench_io_size = 4096;
   }
+  if ((bench_io_size > 100*1024*1024) || (bench_io_size < 1))
```

Comment:
@dillaman 
my cluster has three nodes, each node configured 64G memory，
i ran "bench-write" with "--io-size 1G" and without other parameter。